app/styles/component/_smart-banner-test.scss

```diff
@@ -0,0 +1,66 @@
+.smart-banner-test {
+	align-items: center;
+	background: #f2f5f5;
+	background-size: auto 100%;
+	display: flex;
+	height: 85px;
+	-webkit-font-smoothing: antialiased;
+	width: 100%;
+  	justify-content: flex-start;
```

Comment:
afair we're writing css' props alphabetically, same applies to rules below

Reply:
👍